samples/unit-tests/gantt/xrange-series/demo.js

```diff
@@ -255,7 +259,15 @@
         assert.notEqual(
             chart.tooltip.label.element.textContent.indexOf('Monday'),
             -1,
-            'The tooltip header should container formatted date (#9301)'
+            'The tooltip header should contain a formatted date (#9301)'
+        );
+
+        chart.series[0].update({ color: 'red' });
+        console.log(chart.series[0].points[0].color);
```

Comment:
`console.log` 😬

Reply:
woops, my bad!! i cannot understand how this passes pre-commit 🤔

Reply:
removed